core/controller/src/main/scala/whisk/core/entitlement/Entitlement.scala

```diff
@@ -67,7 +71,8 @@ protected[core] object EntitlementProvider {
     WhiskConfig.actionInvokePerMinuteLimit -> null,
     WhiskConfig.actionInvokeConcurrentLimit -> null,
     WhiskConfig.triggerFirePerMinuteLimit -> null,
-    WhiskConfig.actionInvokeSystemOverloadLimit -> null)
+    WhiskConfig.actionInvokeSystemOverloadLimit -> null,
+    WhiskConfig.runtimesNamespaceWhitelist -> "{}")
```

Comment:
this will go away (in favor of using pureconfig); you could also move all these into pureconfig (or not - could wait for a separate PR)

Reply:
I see. So the main difference here was the support for env vars, and not wanting to add a secondary config mechanism (without realizing this was the old mechanic)